static/docs/user-guide/autocomplete.md

```diff
@@ -29,6 +29,23 @@ Depending on what you typed on the command line so far, it completes:
 - Arguments for selected options. For example, `dvc repro` completes with stage
   files to reproduce.
 
+Depending upon the availability of both Bash and Zsh on your system, follow the
+steps given below to Configure Bash and/or Zsh.
+
+You can view shells available on your system by:
+
+```dvc
+    $ cat /etc/shells
+
+    # /etc/shells: valid login shells
+      /bin/sh
+      /bin/bash
+      /bin/rbash
+      /bin/dash
+```
+
+For instance, here since Zsh is unavailable, you only need to configure Bash.
```

Comment:
this change makes sense, but it's not exactly how it works though. Usually engineers are using the same shell they prefer. Either zsh or bash. It's very rare that you need to install it for both at once. What would be useful here is to mention a command that you can run to check what shell is active now. For less experienced users (who don't care is it a bash or zsh) it might save some time.